hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMRecoverLeaseRequest.java

```diff
@@ -101,6 +103,7 @@ public OMRecoverLeaseRequest(OMRequest omRequest) {
   }
 
   @Override
+  @DisallowedUntilLayoutVersion(HBASE_SUPPORT)
```

Comment:
`OMRecoverLeaseRequest` is already on master. Disabling it here would actually be a step backwards. What are we trying to prevent in this case?

Reply:
We never advertise hsync and lease recovery APIs and hsync is disabled by default in Ozone 1.4.0, so it shouldn't cause a regression.
These annotations are used to prevent undefined behavior if a downgrade is necessary.